docs/en/latest/upgrade.md

```diff
@@ -21,6 +21,77 @@ title: Upgrade Guide
 #
 -->
 
+## Upgrade using Helm chart
+
+Note: Before upgrading APISIX Ingress, you need to update the corresponding CRD resource first, k8s will automatically replace it with the default CRD resource version, incompatible items will be discarded, and its configuration needs to be updated to the current version.
+
+### Operating Step
+
+1. Update Helm repo
+
+Before upgrading, you need to update the helm repo to ensure that the resources in the repo are up to date.
+
+```sh
+helm repo update
+```
+
+2. Upgrade CRDs
+
+When the CRD exists, Helm Chart will not automatically update the CRD when upgrading or installing, so you need to update the CRD resource yourself
+
+- Using Helm (Helm version >= 3.7.0)
+
+```sh
+helm show crds apisix/apisix-ingress-controller | kubectl apply -f -
+```
+
+> If the Helm version does not support it, you need to obtain it from the [apisix-helm-chart](https://github.com/apache/apisix-helm-chart) repository.
+> Directory: `charts/apisix-ingress-controller/crds/customresourcedefinitions.yaml`
+>
+> ```sh
+> kubectl apply -f  https://raw.githubusercontent.com/apache/apisix-helm-chart/apisix-0.11.1/charts/apisix-ingress-controller/crds/customresourcedefinitions.yaml
+> ```
+
+3. UpgradeAPISIX Ingress
+
+Just as an example, the specific configuration needs to be modified by yourself.
+
+```sh
+helm upgrade apisix apisix/apisix \
+  --set gateway.type=NodePort \
+  --set ingress-controller.enabled=true \
+  --namespace ingress-apisix \
+  --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix
+```
+
+### Precautions
+
+It is recommended not to upgrade across major versions.
+
+#### 1.4 -> 1.5 -> 1.6
```

Comment:
What about `1.4 to 1.6`? We just need to know the minimum and maximum versions.

Reply:
no, I don't think we can upgrade to v1.6 from v1.4 directly.